.github/workflows/build-images-workflow-run.yml

```
@@ -204,19 +217,20 @@ jobs:
           Source Sha: ${{ needs.cancel-workflow-runs.outputs.sourceHeadSha }}
           Merge commit Sha: ${{ needs.cancel-workflow-runs.outputs.mergeCommitSha }}
           Target commit Sha: ${{ needs.cancel-workflow-runs.outputs.targetCommitSha }}
+        run: printenv
+      - name: Selective checks
+        id: selective-checks
+        env:
+          EVENT_NAME: ${{ needs.cancel-workflow-runs.outputs.sourceEvent }}
+          MERGE_COMMIT_SHA: ${{ needs.cancel-workflow-runs.outputs.mergeCommitSha }}
         run: |
-          printenv
-      - name: Set versions
-        id: versions
-        run: |
-          . ./scripts/ci/libraries/_script_init.sh
-
-          initialization::ga_output python-versions \
-              "$(initialization::parameters_to_json "${CURRENT_PYTHON_MAJOR_MINOR_VERSIONS[@]}")"
-          initialization::ga_output default-python-version "${CURRENT_PYTHON_MAJOR_MINOR_VERSIONS[0]}"
-
-          initialization::ga_output all-python-versions \
-              "$(initialization::parameters_to_json "${ALL_PYTHON_MAJOR_MINOR_VERSIONS[@]}")"
+          if [[ ${EVENT_NAME} == "pull_request" ]]; then
```

```suggestion
          if [[ "${EVENT_NAME}" == "pull_request" ]]; then
```

just as best-practise: https://github.com/koalaman/shellcheck/wiki/SC2086

---

Not really  -  inside the '[[ ]]' the left hand side part of variable does not need quoting and it shellheck does not complain https://unix.stackexchange.com/questions/68694/when-is-double-quoting-necessary 

I actually every were use convention that I omit the quotes everywhere in bash in the left-siide hand of [[ ]]. This makes it a bit less verbose.

---

yea, as I said in another comment, not strictly required but having quotes, is harmless and just for uniformity since we have got quotes in other places ([example](https://github.com/apache/airflow/blob/master/scripts/docker/install_mysql.sh#L21)) :) but yeah feel free to ignore it :) .